app/assets/javascripts/select-kit/addon/components/composer-actions.js

```diff
@@ -100,6 +100,7 @@ export default DropdownSelectBoxComponent.extend({
   },
 
   @discourseComputed("seq")
+  // even though we don't use seq in the method we still need it to make sure UI updates correctly
```

Comment:
No sorry i wasn't clear:

- you need it in @discourseComputed so it tracks updates on seq
- but you don't need it on content(seq) if you don't use seq

Reply:
Oh, all right 👍 .

Reply:
Then it'll be pretty obvious without comment why we need it, so I will remove it.